docs/guides/other-guides/monitor-ngrok/monitor-ngrok.md

```diff
@@ -0,0 +1,312 @@
+
+You can use policies to request passwords, block malicious traffic, route requests, rewrite URLs, and respond with custom content. If you use a custom permanent domain name (called a [Cloud Endpoint](https://ngrok.com/docs/universal-gateway/cloud-endpoints)) on ngrok, you can set a policy for every agent that uses that domain. Otherwise, for temporary [Agent Endpoints](https://ngrok.com/docs/universal-gateway/agent-endpoints), you can set a Traffic Policy inside each agent individually.
+
+Let's look at rate limiting as an example of a Traffic Policy. Rate limiting is one of the many [Traffic Policy actions](https://ngrok.com/docs/traffic-policy/actions/rate-limit) available. You can rate limit your login page to make password guessing tedious for hackers, or rate limit SMS requests to prevent automated attacks that try to incur massive costs to your business.
+
+At this stage of the guide, you could send unlimited requests to the test API you've created. Let's change that to allow only one request a minute.
+
+In order to test policies, you need to be able to reuse the same URL, which isn't possible if you keep using the temporary URLs that ngrok generates each time you restart an agent. So let's create a permanent URL:
+
+- Browse to the **Domains** page in the ngrok sidebar. If you're using a free account, register for your free domain with a random name, like `massive-pelican-arguably.ngrok-free.app`. If you're using a paid account, create a new domain for this test.
+- Next, you need to stop and restart the ngrok container using a traffic policy.
+- Stop the ngrok Docker container in the terminal with <kbd>Ctrl</kbd><kbd>+</kbd><kbd>C</kbd>.
+- Create a file named `policy.yml` with the code below.
+
+  ```yml
+  on_http_request:
+    - actions:
+      - type: rate-limit
+        config:
+          name: 1 request per minute
+          algorithm: sliding_window
+          capacity: 1
+          rate: 1m
+          bucket_key:
+            - "hasReqHeader('host') ? getReqHeader('host')[0] : 'unknown'"
+  ```
+
+- Run the agent again with the command below. Replace the authentication token and the URL with your token and URL (using `https`, not `http`).
+
+  ```sh
+  docker run -it --rm --platform=linux/amd64 --network=ngrokTest -v ".:/app" -w "/app" -e NGROK_AUTHTOKEN=Y0urS3cr3t ngrok/ngrok:3.22.0-alpine-amd64 http http://api --traffic-policy-file policy.yml --url https://massive-pelican-arguably.ngrok-free.app
+  ```
+
+- Browse to the API URL and refresh the page a few times. Notice that the page stops responding.
+- On the ngrok Traffic Inspector page, note that the error code **`429`** (rate limit) was returned and that the duration of the request was instant and caused no load on your API.
+
+  ![Rate limit](./img/rateLimit.png)
+
+- Edit the `policy.yml` file and change `capacity` to `10`, then restart the ngrok container.
```

Comment:
Given that we're funneling folks through the API gateway guides first, we can remove these steps and just direct folks to see the impact of the rate limiting Traffic Policy action they've already set up.